source/ptreferential/ptreferential.cpp

```diff
@@ -275,9 +280,85 @@ std::vector<idx_t> manage_odt_level(const std::vector<type::idx_t>& final_indexe
     return final_indexes;
 }
 
+bool keep_vj(const nt::VehicleJourney* vj,
+             const bt::time_period& period) {
+    if (vj->stop_time_list.empty()) {
+        return false; //no stop time, so it cannot be valid
+    }
+    const auto first_departure_dt = vj->stop_time_list.front().departure_time;
```

Comment:
yep it's in the specs (not sure about the rational though)

yep @xlqian we can, but i always thought that taking ref on an int was strange ;)